src/DIPS.Xamarin.UI/Controls/DatePicker/DatePicker.xaml.cs

```diff
@@ -122,17 +131,25 @@ public DateTime MinimumDate
             set => SetValue(MinimumDateProperty, value);
         }
 
+        /// <summary>
+        /// Opens the date picker
+        /// </summary>
+        public void Open()
+        {
+            FormsDatePicker.Focus();
+        }
+
+        /// <summary>
+        /// Boolean value to indicate if the date picker is open
+        /// </summary>
+        public bool IsOpen => FormsDatePicker.IsFocused;

```

Comment:
Where are you supposed to use this? As you can't bind to it?

Reply:
I've used it as a if-check before running `Open()`. But, now that you mention it, this check could be a part of `Open()` method.